### PR TITLE
Add static content to no exposures page

### PR DESCRIPTION
**Pull request by amandabeiner**
#### Description:
Why:
As a user, when I have no recent exposures, I want to see information
from my health authority about how I can stay healthy.

This commit:
Adds information to the NoExposures screen that includes tips on staying
healthy and a link to the health authority's guidelines if the health
authority has provided a link

#### Linked issues:
Part of [Exposure List Screen](https://trello.com/c/QUVhbT54/245-exposure-list-screen)

#### Screenshots:
When HA has link:
Simulator Screen Shot - iPhone 11 - 2020-07-28 at 12 24 30

When HA does not have link:
Simulator Screen Shot - iPhone 11 - 2020-07-28 at 12 42 57

#### How to test:
- Clear all exposures
- Navigate to the exposure history screen
- Expect screen to match screenshot(s) above

### CODE REVIEW

File: src/ExposureHistory/History/NoExposures.tsx [hunk 1]
```diff
@@ -1,21 +1,97 @@
 import React, { FunctionComponent } from "react"
-import { View, StyleSheet } from "react-native"
+import env from "react-native-config"
+import { Linking, View, StyleSheet, TouchableOpacity } from "react-native"
 import { useTranslation } from "react-i18next"
 
 import { RTLEnabledText } from "../../components/RTLEnabledText"
 
 import { Colors, Typography, Spacing, Outlines } from "../../styles"
+import { SvgXml } from "react-native-svg"
```
Comment (johnschoeman) [inline src/ExposureHistory/History/NoExposures.tsx:9]: what do you think about keeping third party dependencies at the top?

File: src/ExposureHistory/History/NoExposures.tsx [hunk 2]
```diff
@@ -1,21 +1,97 @@
...
+    <>
+      <View style={style.container}>
+        <RTLEnabledText style={style.headerText}>
+          {t("exposure_history.no_exposure_reports")}
+        </RTLEnabledText>
+        <RTLEnabledText style={style.subheaderText}>
+          {t("exposure_history.no_exposure_reports_over_past")}
+        </RTLEnabledText>
+      </View>
+      <View style={style.card}>
```
Comment (johnschoeman) [inline src/ExposureHistory/History/NoExposures.tsx:28]: what do you think about breaking this card into a subcomponent?  I would keep it in the same file, if so.

File: src/ExposureHistory/History/NoExposures.tsx [hunk 3]
```diff
@@ -1,21 +1,97 @@
...
   const { t } = useTranslation()
   return (
-    <View style={style.container}>
-      <RTLEnabledText style={style.headerText}>
-        {t("exposure_history.no_exposure_reports")}
-      </RTLEnabledText>
-      <RTLEnabledText style={style.subheaderText}>
-        {t("exposure_history.no_exposure_reports_over_past")}
-      </RTLEnabledText>
+    <>
```
Comment (johnschoeman) [inline src/ExposureHistory/History/NoExposures.tsx:19]: did we check this on a small format screen? will this need to be a scroll view or does everything fit?

**Comment by devinjameson** [comment]
Oops we added a commit here, meant to do it on another branch. Hold please.

**Pull request merged**